meson.build

```diff
@@ -142,4 +142,10 @@ if use_pythran
   xsimd_dep = dependency('xsimd', required: false)
 endif
 
+libnpymath_dep = dependency(
```

Comment:
Maybe this deserves a comment like:
```
# We don't expect `libnpymath` to be packaged separately, but looking for a standalone
# install is useful in case of local development (don't have to touch the git submodule)
```

At least I assume that that's the intent. I haven't checked the `libnpymath` repo yet. Are there pkg-config files for it, or is that planned? If not, the above doesn't work and hence we don't need to try looking for the dependency.

Reply:
In my mind this was to ensure that this continues to work even if `libnpymath` is shipped another way. Right now, it does not ship any `pkg-config` files, but it could in the future. Either way I'll remove this and go with the `subproject` call for now and maybe change it in the future in case we do something else.

Reply:
That sounds good, thanks. It's easy enough to change back later once someone actually needs this.